apps/site/pages/docs/api-extensions.mdx

```diff
@@ -317,30 +324,149 @@ export const fragment = gql`
     <details>
       <summary>Query: `ClientProductsQuery`</summary>
       <ExtendableQueryTable>
+        <td>ClientProducts</td>
         <td>Client</td>
         <td>search</td>
         <td>
-          PLP + Pages that use `ProductShelf`, and `ProductTiles` components.
+          `PLP`, `Search Page`, and pages that use `ProductShelf`, and `ProductTiles` components.
         </td>
         <td>
-          - In the hook `useProducts()` from `GalleryPage` from `PLP`.
+          - In the hook `usePageProductsQuery()` from `PLP` and `Search Page`.
           <br />
           - In the hook `useProductsPrefetch()` to prefetch the previous (prev)
-          and next (next) page from the current `PLP` page.
+          and next (next) page from the current `PLP` or `Search Page`.
           <br />
           - In the hook `useProductsQuery()`, in `ProductShelf`, `ProductTiles`
           components, that can be used on multiple pages.
           <br />
         </td>
         <td>
-          General products data (StoreProduct) with the `totalCount` from
-          StorePageInfo.
+          General products data ([StoreProduct](https://v1.faststore.dev/reference/api/objects#storeproduct)) with the `totalCount` from [StorePageInfo](https://v1.faststore.dev/reference/api/objects#storepageinfo).
         </td>
         <td>
-          Frontend data from the `useLocalizedVariables()` and `useSession()`
-          hooks.
+          Frontend data from the `useLocalizedVariables()` and `useSession()` hooks.
         </td>
       </ExtendableQueryTable>
     </details>
   </li>
 </ul>
+
+---
+
+## How to consume FastStore API Extension
+
+
+Once you define which new fields will be exposed by the FastStore API Extension, then determine the GraphQL fragments where these fields will be used.
+Finally, it's time to know how to consume the new fields in your [New Sections](/docs/building-sections/creating-a-new-section) or [Custom Overridable Components](/docs/building-sections/overriding-components-and-props).
+
+It's important to know that FastStore exposes the data that comes from FastStore API along with FastStore API Extensions inside a provider called `PageProvider`.
+This data comes as a `context` in `PageProvider`, and you can use some hooks to access it:
```

Comment:
```suggestion
This data comes as a `context` in `PageProvider`. FastStore provides three specific hooks (`usePDP`, `usePLP`, and `useSearchPage`) and one generic (`usePage`) that you can use to access the data. If the component or section will always appear on a single page type, we encourage you to use the specific page hook. If it can be used on multiple page types, use the generic one and narrow it using the type assertion functions provided (`isPDP`, `isPLP`, `isSearchPage`).
```

I'd make this paragraph longer and remove all the code snippets below, as they mostly add noise IMO.